src/Lumi/Components2/Box.purs

```diff
@@ -25,3 +26,9 @@ box =
             , className: props.className
             , css: toCSS theme props Styles.Box.box
             }
+
+row :: LumiComponent BoxProps
+row = box <<< Styles.Box._row
```

Comment:
The key benefit of this change for me is not the syntactic cleanliness — although that's a big one for me too —, but the improved composability: the fact that, now, preapplying prop modifiers to `LumiComponent`s result in `LumiComponent`s.